doc/md/cloud/getting-started.md

```diff
@@ -42,15 +42,17 @@ If you already have the Atlas GitHub action set up, you may skip step 4. In step
 `ariga-token: ${{ secrets.ARIGA_TOKEN }}` to your yaml file.
 :::
 
-1. From the Settings page, generate an access token under 'Tokens'.
-2. On your GitHub repo, under the 'Settings' section, click on 'Secrets' > 'Actions' to create a new repository secret.
+1. From the Settings page, under 'Bots', click on "Create Bot" to create a new CI bot.  
+The CI bot will be used to connect your GitHub and allow Atlas Cloud to lint your pull requests.
```

Comment:
```suggestion
```

Reply:
I think I would add this sentence higher up, under 'Connecting to Atlas with a GitHub Action'
Something like 'In order to connect to the Atlas GitHub Action, we will create a bot user. Bot users are used to ..."